connectors/salesforce/element-templates/salesforce-connector.json

```
@@ -0,0 +1,434 @@
+  },
+  "documentationRef": "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/rest/",
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:Task"
+  ],
+  "elementType": {
+    "value": "bpmn:ServiceTask"
+  },
+  "groups": [
+    {
+      "id": "authentication",
+      "label": "Authentication"
+    },
+    {
+      "id": "endpoint",
+      "label": "Operation"
+    },
+    {
+      "id": "input",
+      "label": "Payload"
+    },
+    {
+      "id": "timeout",
+      "label": "Connect timeout"
+    },
+    {
+      "id": "output",
+      "label": "Response mapping"
+    },
+    {
+      "id": "errors",
+      "label": "Error handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:http-json:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "label": "Type",
+      "id": "authenticationType",
+      "group": "authentication",
+      "description": "Choose the authentication type",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Bearer token",
+          "value": "bearer"
+        },
+        {
+          "name": "OAuth 2.0",
+          "value": "oauth-client-credentials-flow"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      }
+    },
+    {
+      "label": "Salesforce Base URL",
+      "group": "endpoint",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "baseUrl"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "pattern": {
+          "value": "^(=|http://|https://|secrets).*$",
+          "message": "Must be a http(s) URL."
+        }
+      }
+    },
+    {
+      "id": "salesforceOperationType",
+      "label": "Salesforce Operation Type",
+      "description": "The kind of operation to perform with the Salesforce REST API",
+      "type": "Dropdown",
+      "group": "endpoint",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "salesforceInteractionType"
+      },
+      "choices": [
+        {
+          "name": "sObject CRUD",
+          "value": "sObject"
+        },
+        {
+          "name": "SOQL Query",
+          "value": "soqlQuery"
+        }
+      ]
+    },
+    {
+      "label": "Interaction Type",
+      "id": "interactionType",
+      "group": "endpoint",
+      "description": "Choose the kind of interaction you want to perform",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Get Object",
+          "value": "get"
+        },
+        {
+          "name": "Create Object",
+          "value": "post"
+        },
+        {
+          "name": "Update Object",
+          "value": "patch"
+        },
+        {
+          "name": "Delete Object",
```

Should follow the same style as other connectors:

```suggestion
          "name": "Delete object",
```